### ISSUE TITLE
finalize data model

### ISSUE THREAD

**Issue by egonw** [bridgedb/tiwid#1]
At this moment I do not have clear yet what the data model is going to be. These things come to mind at this moment:

- died identifier
- when the identifier died
- type of identifier: dead, zombie, ghost
- next of kin

**Comment by Chris-Evelo** [comment]
I think you would also want to know *why* the identifier died. Things like "gene turns out not to be coding" or "green turns out to be two separate genes" are really different things. (And in both cases we might actually want to keep a record even though the identifier was removed from the most recent database since you could still do things like: "map variants mapped to the now longer gene area to the chromosome location", decide that gene expression for a cluster should belong to one of two genes and possibly you could decide which one based on reporter sequence)

**Comment by egonw** [comment]
Yes, indeed. Once we have one or more data sources that mention this, we can start modelling this. WikiPathways has similar things, like "merged content into". Ensembl does not seem to provide this information.

**Comment by cthoyt** [comment]
Would be nice to add the curator ORCID identifier to keep track of attributions

**Comment by matentzn** [comment]
@egonw can I take a look at the current overall data model? This seems to be all very related to ontology-metadata and mapping metadata efforts we are trying to reconcile across the board..

**Comment by cthoyt** [comment]
@matentzn the data model is currently 3 columns in a TSV file - see https://github.com/bridgedb/tiwid/blob/main/data/hgnc.symbol.tsv as an example

**Comment by matentzn** [comment]
Thank you @cthoyt! I thought this was related to the more broad birdgedb data models, not just tiwid, but makes sense now.

**Comment by egonw** [comment]
@matentzn, let's start a discussion here: https://github.com/bridgedb/BridgeDb/issues/216

**Comment by egonw** [comment]
Oh, and with SSSOM in mind, this format may have mappings but these are optional: there is no clear mapping necessarily.

**Comment by matentzn** [comment]
Ok, got it :) I remember looking at bridgedb in the past, and talking to @Chris-Evelo about adopting sssom for any mappings, but I did not really follow up with that!
